pkg/crc/preflight/preflight_linux.go

```diff
@@ -146,21 +151,34 @@ func fixVsock() error {
 	if err != nil {
 		return err
 	}
-	_, _, err = crcos.RunWithPrivilege("modprobe vhost_vsock", "modprobe", "vhost_vsock")
+
+	udevRule := `KERNEL=="vsock", MODE="0060", OWNER="root", GROUP="libvirt"`
```

Comment:
why '==' for kernel and '=' for the others?

Reply:
the kernel is a match key and  is for matching with the device name provided by the kernel, that's the criteria to apply the rule so its a comparision "==" while the others we are assigning the values to the keys so "="